tests/unit/keyspace.tcl

```diff
@@ -71,6 +71,12 @@ start_server {tags {"keyspace"}} {
         append res [r exists emptykey]
     } {10}
 
+    test {MEXISTS} {
+        r set a{t} 1
```

Comment:
It is best to delete c{t} d{t} to prevent it from being affected by other test cases
```suggestion
        r del a{t} b{t} c{t} d{t}
        r set a{t} 1
```

Reply:
@enjoy-binbin Thank you!